sitemap/sitemap.go

```diff
@@ -72,25 +74,28 @@ type URLset struct {
 	URL        []URL    `xml:"url"`
 }
 
+var testFolderPath string
+
 func (repository *SitemapRepository) GenerateSitemap(c *gin.Context) {
 	baseUrl := c.Query("baseUrl")
 
 	sitemapUrls := []URL{
 		{Loc: baseUrl, Priority: `1`},
-		{Loc: baseUrl + `/contact`, Priority: `0.9`},
-		{Loc: baseUrl + `/about`, Priority: `0.9`},
+		{Loc: baseUrl + "/resources", Priority: `0.9`},
 	}
 
-	// add portfolios
-	sitemapUrls = addPortfolios(baseUrl, sitemapUrls)
-
-	// add all jobs
-	sitemapUrls, err := repository.addJobs(baseUrl, sitemapUrls)
+	var folderPath string
+	if testFolderPath != "" {
```

Comment:
Where `testFolderPath` is setting?

Reply:
its name is relevant as its used in test file and set there